ui/apps/platform/src/Containers/MainPage/Body.tsx

```diff
@@ -138,66 +125,13 @@ function Body({ hasReadAccess, isFeatureFlagEnabled }: BodyProps): ReactElement
                 <Switch>
                     <Route path="/" exact render={() => <Redirect to={dashboardPath} />} />
                     <Route path={mainPath} exact render={() => <Redirect to={dashboardPath} />} />
-                    <Route path={dashboardPath} component={AsyncDashboardPage} />
-                    <Route path={networkPath} component={AsyncNetworkGraphPage} />
-                    <Route path={violationsPath} component={AsyncViolationsPage} />
-                    <Route path={compliancePath} component={AsyncCompliancePage} />
-                    <Route
-                        path={complianceEnhancedBasePath}
-                        component={AsyncComplianceEnhancedPage}
-                    />
-                    <Route path={integrationsPath} component={AsyncIntegrationsPage} />
-                    <Route path={policyManagementBasePath} component={AsyncPolicyManagementPage} />
                     {/* Make sure the following Redirect element works after react-router-dom upgrade */}
                     <Redirect exact from={deprecatedPoliciesPath} to={policiesPath} />
```

Comment:
Unrelated to this PR, but do we have a policy on how long we keep redirects for deprecated routes in the app?

Reply:
That is a good subject for discussion.

Especially because this specific case did not easily translate from React Router 5 to 6 in my quick-and-dirty effort to convert.